tests/conftest.py

```diff
@@ -135,7 +135,9 @@ def pytest_addoption(parser):
     ########################
     parser.addoption("--deep_clean", action="store_true", default=False,
                      help="Deep clean DUT before tests (remove old logs, cores, dumps)")
-
+    parser.addoption("--enable_container_autorestart", action="store_true", default=False,
```

Comment:
This option is to skip the 'disable_container_autorestart' fixture. Can you rename this accordingly and also modify the help to better state its purpose?

Reply:
Update the description and name to `skip_fixture_disable_container_autorestart`. Thanks.